### PR TITLE
Create Blank panel for spacing and stuff

### PR DESCRIPTION
**Pull request by 2metres**
**User-Facing Changes**
Screen Shot 2022-10-22 at 5 41 05 am

**Description**
Adds blank panel with optional foxglove logo.

Co-authored by insomina 😵‍💫

### CODE REVIEW

**Review by jtbandes — CHANGES_REQUESTED** [review]
This should be an extension.

**Comment by jtbandes** [comment]
However, it looks like a nice splash screen. Maybe we should add a splash screen to the app 😀

**Comment by 2metres** [comment]
@jtbandes I tried to start with an extension but couldn't figure it out. Can we pair next week on making it one?

File: packages/studio-base/src/panels/Blank/index.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+export const Default = () => <Blank />;
+
+Default.parameters = {
+  colorScheme: "both-column",
+};
+
+export const NoLogo = () => <Blank />;
+
+NoLogo.parameters = {
```
Comment (2metres) [inline packages/studio-base/src/panels/Blank/index.stories.tsx:31]: @foxymiles can you help me to pass the config to the storybook?

**Comment by jtbandes** [comment]
Sure I can help out with the extension. Did you try starting with one of the templates/examples e.g. https://github.com/foxglove/create-foxglove-extension/tree/main/examples/panel-settings ?

**Comment by 2metres** [comment]
I was not aware that existed, Maybe we should make this a blog post as well. Simplist panel I can imagine.

File: packages/studio-base/src/panels/Blank/index.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+export function Default(): JSX.Element {
+  return <Blank />;
+}
+Default.parameters = { colorScheme: "both-column" };
+
+export function NoLogo(): JSX.Element {
+  return <Blank />;
+}
+NoLogo.parameters = { colorScheme: "both-column" };
```
Comment (2metres) [inline packages/studio-base/src/panels/Blank/index.stories.tsx:32]: @foxymiles can you help me to pass the config to the storybook?

**Comment by 2metres** [comment]
Closed in favor of adding as an Extension

**Pull request closed without merging**